src/AtomBeautify.ts

```diff
@@ -3,10 +3,10 @@ import * as Atom from "atom";
 import beautifiers from "./beautifiers";
 import { CompositeDisposable, Disposable } from "atom";
 import Config from "./config";
-import * as Promise from "bluebird";
 import * as path from "path";
 import * as _ from "lodash";
 const pkg = require("../package");
+const cosmiconfig = require("cosmiconfig");
```

Comment:
Should be using `import`.

`@types/cosmiconfig` exists: https://yarnpkg.com/en/package/@types/cosmiconfig

Reply:
I ended up needing to add the typings myself. Appears @types/cosmiconfig may be broken.

- https://github.com/Unibeautify/vscode/blob/master/src/%40types/cosmiconfig/index.d.ts
- https://github.com/Unibeautify/vscode/blob/master/tsconfig.json#L13-L19